### PR TITLE
MA-3231:Updated SNS Topic lib file to add and get SNS Topic's Attribu…

### PR DESCRIPTION
**Pull request by bhargavapratik**
MA-3231:Updated SNS Topic lib file to add and get SNS Topic's Attributes.

### CODE REVIEW

**Review by chandramishra — APPROVED** [review]
LGTM

File: aws/topic.js [hunk 1]
```diff
@@ -363,6 +363,49 @@ console.log(params);
...
+    if (callback) {
+      var sns = me.findService(input);
+      sns.setTopicAttributes(params, callback);
+     if (err) console.log(err, err.stack); // an error occurred
+    else     console.log(data);           // successful response
+      return;
+    }
+
+    var sns = me.preRun(self, input);
+    sns.setTopicAttributes(params, me.callback);
```
Comment (RajkumarShivage) [inline aws/topic.js:387]: can we handle the exception here?

Comment (bhargavapratik) [reply aws/topic.js:387]: It is handeled in if block above from line 378 to 384

Comment (RajkumarShivage) [reply aws/topic.js:387]: ignore

**Review by RajkumarShivage — APPROVED** [review]
LGTM

**Pull request merged**